cis_v130/section_4.sp

```diff
@@ -180,9 +202,10 @@ control "cis_v130_4_3_1" {
   documentation = file("./cis_v130/docs/cis_v130_4_3_1.md")
 
   tags = merge(local.cis_v130_4_3_common_tags, {
-    "cis_item_id" = "4.3.1"
-    "cis_level"   = "1"
-    "cis_type"    = "automated"
+    cis_item_id = "4.3.1"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "Azure/DatabaseForPostgreSQL"
```

Comment:
"Azure/DatabaseForPostgreSQL" or "Azure/PostgreSQL" with [reference to](https://github.com/turbot/steampipe-plugin-azure/blob/be913eacb88adbab93ca6749bb805d11731785d0/azure/table_azure_postgresql_server.go#L11)